### PR TITLE
Updating Readme.md with circleci status badges

### PR DESCRIPTION
**Pull request by algojack**
since we moved most of our builds to circleci i think it's time to update the status badges in the readme.md to reflect that

### CODE REVIEW

**Review by tsachiherman — DISMISSED** [review]
Looks good to me.

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Build Status](https://travis-ci.com/algorand/go-algorand.svg?branch=master)](https://travis-ci.com/algorand/go-algorand)
+| rel/stable [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fstable.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fstable) | rel/beta [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fbeta.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fbeta) | rel/nightly [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fnightly.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fnightly) |
+| --- | --- | --- |
```
Comment (winder) [inline README.md:2]: I didn't notice the weird alignment when Jack made this change on Indexer, but now I can't unsee it... for your consideration:
```suggestion
<center>
| rel/stable </br> [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fstable.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fstable) | rel/beta  </br> [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fbeta.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fbeta) | rel/nightly  </br> [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fnightly.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fnightly) |
| --- | --- | --- |
</center>
```

Comment (algojack) [reply README.md]: thank you! added.

**Review by winder — APPROVED** [review]

**Review by egieseke — APPROVED** [review]
Looks good.

File: README.md (hunk 1 above)
Comment (algobarb) [reply README.md]: I'm not sure if it's just my browser, but the centered changes look like the screenshot (below) for me when I look at the README.md.
image

Comment (algobarb) [reply README.md]: Remove the `<center></center>` and it will look like the screenshot Will had in the "after" image.

File: README.md [hunk 2]
```diff
@@ -1,4 +1,7 @@
-[![Build Status](https://travis-ci.com/algorand/go-algorand.svg?branch=master)](https://travis-ci.com/algorand/go-algorand)
+<center>
```
Comment (algobarb) [inline README.md:1]:
```suggestion

```

File: README.md [hunk 3]
```diff
@@ -1,4 +1,7 @@
...
+| rel/stable </br> [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fstable.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fstable) | rel/beta  </br> [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fbeta.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fbeta) | rel/nightly  </br> [![CircleCI](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fnightly.svg?style=svg)](https://circleci.com/gh/algorand/go-algorand/tree/rel%2Fnightly) |
+| --- | --- | --- |
+</center>
```
Comment (algobarb) [inline README.md:4]:
```suggestion

```

**Review by algobarb — COMMENTED** [review]
remove the center tags and leave the `</br>` tags

**Pull request merged**